api/job_events_test.go

```diff
@@ -94,26 +94,22 @@ func TestJobEventsSubscribe(t *testing.T) {
 				time.Sleep(20 * time.Millisecond)
 			}
 
-			result := apitest.New().
```

Comment:
Is there a particular reason we need to change the test? From an outside POV there should be no difference.

Reply:
Yes, internally `apitest` uses a mock which doesn't implement the `http.Hijacker` interface (and hence the test fails). That's why I've had to introduce the `MockResponseRecorder`.